crates/astria-conductor/src/bin/astria-conductor.rs

```diff
@@ -118,7 +118,12 @@ async fn run() -> Result<()> {
             _ = stop_rx.changed() => {
                 info!("shutting down conductor");
                 if let Some(e) = driver_tx.send(DriverCommand::Shutdown).err() {
-                    error!("error sending Shutdown command to driver: {}", e);
+                    error!(
+                        actor_name = "driver",
+                        error.msg = %e,
+                        error.cause = ?e,
```

Comment:
will this end up printing the most recent error twice (since it prints it as both display + debug)?

Reply:
I think not for this case because `SendError` has different implementations of display and debug, but I think this would cause duplicate logs for the other places I am using this.

Reply:
@SuperFluffy Do you have any insight on when to use this syntax? What should it look like for other errors?